frontend/packages/schema-editor/src/components/NodePanel/HeadingRow/HeadingRow.module.css

```diff
@@ -0,0 +1,22 @@
+.root {
+  --gap: var(--fds-spacing-1);
+  align-items: center;
+  display: flex;
+  padding: var(--gap) 0;
+  gap: var(--gap);
+}
+
+.root.selected .headingButton {
+  background-color: var(--fds-semantic-surface-action-first-no_fill-active);
+}
+
+.heading {
+  display: contents;
+}
+
+.heading .headingButton {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
```

Comment:
Are these variables supposed to be here? 😄

Reply:
In this case we want button width the height and font corresponding to the one with `size='large'`, but with spacings corresponding to `size='small'` to keep it aligned with the buttons in the tree view. There are two ways to accomplish this:
- Setting `size='small'` and overriding height and font size
- Setting `size='large'` and overriding margins and paddings
I chose the first approach since height and font size are more stable properties, and it'll not be such a big deal should they change.